src/frontend/src/pages/myfeeds.js

```diff
@@ -14,6 +14,28 @@ const useStyles = makeStyles(theme => ({
 
 export default function MyFeeds() {
   const classes = useStyles();
+  const [feedAuthor, setFeedAuthor] = useState('');
+  const [feedUrl, setFeedUrl] = useState('');
+
+  function handleAuthorChange(author) {
+    setFeedAuthor(author);
+  }
```

Comment:
I'm curious as to what use cases might depend on this functionality (since the author of every added feed might be assumed to be the name of the user who created the feed). Perhaps this would be useful for administrative purposes?
<hr>

Reflecting back on initial prototype, which features a disabled `author` text field: I suppose the sole purpose of that field, as I envisioned it, was to act as a read-only reminder to the current user of what name would be associated with their feeds. That sole purpose, however, is hardly substantial, and may not fully merit keeping the `author` field in there (unless the field become mutable, e.g. as suggested by this PR).

Reply:
You're right, I just needed to disable it for now so it doesn't conflict with the API